AnagramSolver.BusinessLogic/Classes/AnagramSolver.cs

```diff
@@ -8,38 +8,46 @@ namespace AnagramSolver.BusinessLogic.Classes
 {
     public class AnagramSolver : IAnagramSolver
     {
-        WordRepository word = new WordRepository();
-        public List<Anagram> GetAnagrams(string command)
+        WordRepository wordRepository = new WordRepository();
+        public List<Anagram> GetAnagrams(string command, int minLength, string filePath)
         {
-            var allWords = word.GetAllWords();
+            var allWords = wordRepository.GetAllWords(filePath, minLength);
 
             // var exists = allWords.FirstOrDefault(p => p.Equals("sula"));
 
             var newList = new List<Anagram>();
 
-            var commandChars = command.ToCharArray();
-            Array.Sort(commandChars);
+            command = command.Replace(" ", string.Empty);
 
-            foreach (var word in allWords)
+            var commandChars = command.ToList();
+            var remainingChars = commandChars;
+            var result = string.Empty;
+
+            sentenceMaker(remainingChars);
+            
+            void sentenceMaker(List<char> remainingChars)
```

Comment:
Naming convention. Methods should start with capital letter (https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions)

Nested methods has it's drawbacks. You should experience them when writing tests for this class.